apps/base-docs/tutorials/docs/5_shopify-storefront-commerce.md

```diff
@@ -0,0 +1,119 @@
+
+### Link Coinbase Commerce to your Shopify store
+
+![shopify-install-commerce.gif](../../assets/images/shopify-storefront-commerce/shopify-install-commerce.gif)
+
+Once, you've created a Coinbase Commerce Managed account you can now add the plugin to your Shopify store.
+
+To add Coinbase Commerce as a payment method on your Shopify store, start by navigating to your admin page at `https://admin.shopify.com/store/<YOUR-STORE-NAME>`.
+Once there, click on Settings , located in the bottom left panel of the screen (or at `https://admin.shopify.com/store/<YOUR-STORE-NAME>settings/general`). Then select the “Payments” tab.
+Proceed by clicking “Add a payment method” and choose “Search by provider.” In the search field, type in “Coinbase” and select “Coinbase Commerce” from the results.
+Click "Install" and you should be redirected to Coinbase Commerce with aprompt log into your Coinbase account. Finally, click "Activate" to enable the Coinbase Commerce plugin.
+
+### Create a Storefront
+
+In this tutorial, we'll guide you through the steps to create a new Hydrogen storefront for your Shopify store. This will allow you to showcase the products you already have in your Shopify account. Make sure you've already created your products by following the [Shopify products guide](https://help.shopify.com/en/manual/online-sales-channels/shop/products-and-collections).
+
+To get started, we'll first clone a Shopify demo store using the Hydrogen framework. This will give us a quick setup to work with.
+
+```bash
+npm create @shopify/hydrogen@latest -- --quickstart
+```
+
+Once the demo store is cloned, navigate into the project directory and start the development server:
+
+```bash
+cd hydrogen-quickstart
+npx shopify hydrogen dev
+```
+
+Next, you'll need to link your Hydrogen project to your Shopify store. This step connects the demo storefront with your Shopify account, allowing you to display your products.
+
+```bash
+npx shopify hydrogen link
+```
+
+To ensure your storefront is properly configured, update the project environment variables. This pulls the necessary settings from your Shopify account into the Hydrogen project.
+
+```bash
+npx shopify hydrogen env pull
+```
+
+Now, let's verify that everything is set up correctly and your site is running. Start the development server again:
+
+```bash
+npx shopify hydrogen dev
+```
+
+Finally, deploy your Hydrogen storefront to Oxygen, Shopify's hosting platform for Hydrogen apps. Run the following script:
+
+```bash
+npx shopify hydrogen deploy
+```
+
+Select **Preview** as the deployment type. After deployment, your terminal should display a URL where you can view your live site, such as:
+
+Your terminal should display a url like:
+`https://hydrogen-quickstart-20c3648d482c7a17d77d.o2.myshopify.dev/`
```

Comment:
I'd also mention how to find it in the web app later:  https://admin.shopify.com/store/b9cbd5-c0/hydrogen